flutter_local_notifications/example/lib/main.dart

```diff
@@ -1017,6 +1039,24 @@ class _HomePageState extends State<HomePage> {
         3, 'Attention', 'Two messages', platformChannelSpecifics);
   }
 
+  Future<void> _showNotificationsWithTag() async {
```

Comment:
Oh, you're right. I wanted to demonstrate the behavior of updating the notification, but the same can be achieved with just ids, so that's unnecessary.